src/Domain/Model/ChannelPricingInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Domain\Model;
+
+use Sylius\Component\Core\Model\ChannelPricingInterface as BaseChannelPricingInterface;
+
+interface ChannelPricingInterface extends BaseChannelPricingInterface, LowestPriceBeforeDiscountAwareInterface
```

Comment:
I wonder if we need this interface, the `LowestPriceBeforeDiscountAwareInterface` should be enough, as it has to be implemented in the end application anyway. In this way, we avoid the potential problem of double implementation of the underlying interface.